tests/messages/test_message_meta.py

```diff
@@ -41,36 +41,36 @@ def fixture_df(
     dataset: DatasetManager,
     index_type: typing.Literal['normal', 'skip', 'dup', 'down',
                                'updown']) -> typing.Union[cudf.DataFrame, pd.DataFrame]:
-    filter_probs_df = dataset["filter_probs.csv"]
+    test_df = dataset["test_dataframe.jsonlines"]
```

Comment:
This is a good new test dataframe. Lets try any use it more places where we were using `filter_probs` before to ensure there are no more problems.